crates/red_knot_python_semantic/src/types.rs

```diff
@@ -556,16 +546,24 @@ pub enum Type<'db> {
 }
 
 impl<'db> Type<'db> {
+    pub const fn annotated_any() -> Self {
+        Self::Any(AnyType::Annotated)
+    }
```

Comment:
nit: I'd be inclined just to call this `Type::any()`

Reply:
Done